tartiflette_plugin_scalars/datetime.py

```diff
@@ -74,7 +74,7 @@ def coerce_output(value: datetime) -> str:
             return value.isoformat()
         if isinstance(value, str):
             try:
-                datetime.fromisoformat(value)
+                return isoparse(value).isoformat()
             except ValueError:
                 raise ValueError(f"DateTime cannot represent value: < {value} >")
             else:
```

Comment:
If return is done in the try, can you also remove the else clause which become unreachable ?

Reply:
fixed